### ISSUE TITLE
Midterm Peer Review ty364

### ISSUE THREAD

**Issue by ghost** [BrandonKates/Deep-RL-for-Polynomial-Combinatorial-Optimization#10]
The project is to evaluate the limitation of Neural Network based methods for the minimum spanning tree problem by build a graph neural network architecture. The algorithm performance will be compared with Kruskal's or Prim's algorithm for solving minimum spanning trees, given a fixed amount of training time, a fixed number of examples, or other limited resources.

What do you like about the proposal?
The research idea is based on a paper around TSP
The research topic is clear and the evaluation method is feasible.

What concerns you?
What is the results? Does the GNN architecture perform better than the Kruskal's or Prim's algorithm for solving minimum spanning trees?

Do you think you could use the results of this study?
The new algorithm is advanced and has reference value.

What other aspects of the question do you think the group should consider?
How does the group achieve the preliminary results comparison with traditional architecture? Is there any benchmarks to quantify the results?

In conclusion, great project. The group developed the project in a clear research logic.
